modin/backends/pandas/query_compiler.py

```diff
@@ -60,6 +60,38 @@ def str_op_builder(df, *args, **kwargs):
     return str_op_builder
 
 
+def _dt_prop_map(property_name):
+    def dt_op_builder(df, *args, **kwargs):
+        dt_s = df.squeeze().dt
+        return getattr(dt_s, property_name).to_frame()
+
+    return dt_op_builder
+
+
+def _dt_map2(property_name):
```

Comment:
What does this do? I don't see it getting used.

Reply:
This is part of my experiments. I have problems with properties `pandas.Series.dt.tz` and `pandas.Series.dt.freq` which returns non-pandas types (`tz` returns `datetime.tzinfo`, for example). And this function are used because this types haven't method `to_frame()`. If you have any ideas tell me, please.